edit/workload/index.vue

```diff
@@ -578,6 +578,20 @@ export default {
         }
       }
 
+      const containerResources = template.spec.containers[0].resources;
+      const nvidiaGpuLimit = containerResources?.limits[GPU_KEY];
+
+      // Though not required, requests are also set to mirror the ember ui
+      if (nvidiaGpuLimit > 0) {
+        containerResources.requests[GPU_KEY] = nvidiaGpuLimit;
+      }
+      if (nvidiaGpuLimit === undefined || nvidiaGpuLimit === 0 ) {
+        try {
+          delete containerResources.limits[GPU_KEY];
+          delete containerResources.requests[GPU_KEY];
```

Comment:
Also, if you remove `GPU_KEY` from either `limits` or `requests` and this leaves those keys with no child properties, you should delete the key as well, rather than leaving an empty key